Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftMD4IoT.yaml

```diff
@@ -11,8 +11,9 @@ References:
 - Title: ASIM Authentication Schema
   Link: https://aka.ms/ASimAuthenticationDoc
 - Title: ASIM
-  Link: https:/aka.ms/AboutASIM
-Description: ASIM Azure Defender for IoT Authentication Parser
+  Link: https://aka.ms/AboutASIM
+Description:  |
+  This ASIM parser supports normalizing Microsoft Defender for IoT logs, produced by the Microsoft Defender for IoT connector, to the ASIM Authentication normalized schema.
```

Comment:
Remove ", produced by the Microsoft Defender for IoT connector". I am not sure the connector actually brings those events (it is a diagnostic setting on their side), and it is not worth checking.